Looms.PoS/Looms.PoS.Application/Features/Order/Commands/UpdateOrder/UpdateOrderRequestValidator.cs

```diff
@@ -0,0 +1,32 @@
+using FluentValidation;
+using Looms.PoS.Application.Models.Requests;
+using Looms.PoS.Application.Utilities.Validators;
+using Looms.PoS.Domain.Enums;
+using Looms.PoS.Domain.Exceptions;
+using Looms.PoS.Domain.Interfaces;
+
+namespace Looms.PoS.Application.Features.Order.Commands.CreateOrder;
+
+public class UpdateOrderRequestValidator : AbstractValidator<UpdateOrderRequest>
+{
+    public UpdateOrderRequestValidator(IDiscountsRepository discountsRepository)
+    {
+        RuleLevelCascadeMode = CascadeMode.Stop;
+
+        RuleFor(x => x.DiscountId!)
+            .MustBeValidGuid()
+            .CustomAsync(async (discountId, _, cancellationToken) => {
+                Console.WriteLine(discountId);
+                Console.WriteLine(Guid.Parse(discountId));
+                if (discountId is not null)
+                {
+                    var discount = await discountsRepository.GetAsync(Guid.Parse(discountId)) ?? throw new LoomsNotFoundException("Discount n found");
```

Comment:
repository throws, this should never throw

Reply:
removed